src/components/Layout/Layout.js

```diff
@@ -126,21 +126,22 @@ const Layout = (props) => {
             <div
               id="appWarningMessage"
               tabIndex="-1"
-              aria-live="polite"
+              aria-live="assertive"
+              aria-atomic="true"
               className="border-1px margin-y-2 padding-2 bg-accent-warm-lighter
                          text-bold text-secondary-vivid react-transition display-none"
             >
               <WarningSharp className="margin-right-2" />
-              <CloseSharp
-                aria-label="close warning alert message"
+              <IconButton  //IconButton does not have aria-label as hidden
                 className="float-right cursor-pointer"
-                onClick={() => hideAppWarning()}
-                onKeyUp={(event) => {
-                  if (event.key === "Enter") {
-                    hideAppWarning();
-                  }
-                }}
+                classes={{ root: classes.closeLogin }}
+                aria-label="close warning alert message"
+                title="close warning alert message"       
+                onClick={() => hideAppError()}
```

Comment:
This should be `hideAppWarning`:
```suggestion
                onClick={() => hideAppWarning()}
```

Reply:
fix this with the last commit